Bottom: adbc5e044ca220d9d21d72e8b804f954a84df8bc
Top:    915e0ee4505ffa5dc789b46752172eee702adae3
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-10-24 22:51:47 -0700

Refresh of rs-iomap

---

diff --git a/src/rsocket.c b/src/rsocket.c
index f0c4c35..29b8c01 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -944,7 +944,6 @@ static int rs_write_direct(struct rsocket *rs, struct rs_iomap *iom, uint64_t of
 {
 	uint64_t addr;
 
-	rs->sseq_no++;
 	rs->sqe_avail--;
 	rs->sbuf_bytes_avail -= length;
